dist/assets/css/primary.css

```diff
@@ -1,10 +1,15 @@
-* { box-sizing: border-box; }
+* {
+    box-sizing: border-box;
+}
 
 html {
-    font-family: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-family: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
```

Comment:
For consistence with HTML, JS and most online CSS documentation including MDN, I'd like to propose we default to double quotes on CSS too.